.github/workflows/ci-e2e.yaml

```diff
@@ -67,13 +68,18 @@ jobs:
           make local-setup
       - name: Deploy MGC
         run: |
-          make docker-build-controller kind-load-controller deploy-controller
+          make docker-build-gateway-controller kind-load-gateway-controller deploy-gateway-controller
           kubectl --context kind-mgc-control-plane -n multicluster-gateway-controller-system wait --timeout=300s --for=condition=Available deployment/mgc-controller-manager
           kubectl --context kind-mgc-control-plane logs --all-containers --ignore-errors deployment/mgc-controller-manager -n multicluster-gateway-controller-system
           kubectl get managedzones -n multi-cluster-gateways mgc-dev-mz-aws -o yaml
           kubectl --context kind-mgc-control-plane -n multi-cluster-gateways wait --timeout=60s --for=condition=Ready managedzone/mgc-dev-mz-aws
           kubectl get managedzones -n multi-cluster-gateways mgc-dev-mz-gcp -o yaml
           kubectl --context kind-mgc-control-plane -n multi-cluster-gateways wait --timeout=60s --for=condition=Ready managedzone/mgc-dev-mz-gcp
+      - name: Deploy Policy Controller
+        run: |
+          make docker-build-policy-controller kind-load-policy-controller deploy-policy-controller
+          kubectl --context kind-mgc-control-plane -n multicluster-gateway-controller-system wait --timeout=300s --for=condition=Available deployment/mgc-policy-controller-manager
+          kubectl --context kind-mgc-control-plane logs --all-containers --ignore-errors deployment/mgc-policy-controller-manager -n multicluster-gateway-controller-system
```

Comment:
I don't think you need the "Deploy Policy Controller" stage since the deployment of MGC includes the gateway controller.

Reply:
not sure I follow. What would reconcile the policies if I don't deploy it? Or do you  mean it is already deployed somehow?

Reply:
You are including it in the deployment  of MGC so it's getting deployed when you run `deploy-gateway-controller`. Making me think you need to add `docker-build-policy-controller kind-load-policy-controller` to that stage, although it did seem to work locally :thinking:

Reply:
OK, it will work because we have the default image set to `quay.io/kuadrant/policy-controller:main`, so we definitely do need to add `docker-build-policy-controller kind-load-policy-controller` to the MGC deploy stage or we will be testing the wrong thing.

Reply:
yep
```
      - name: Deploy
        run: |
          make docker-build-policy-controller kind-load-policy-controller deploy-policy-controller
          make docker-build-gateway-controller kind-load-gateway-controller deploy-gateway-controller
          kubectl --context kind-mgc-control-plane -n multicluster-gateway-controller-system wait --timeout=300s --for=condition=Available deployment/mgc-controller-manager
          kubectl --context kind-mgc-control-plane -n multicluster-gateway-controller-system wait --timeout=300s --for=condition=Available deployment/mgc-policy-controller-manager
          kubectl --context kind-mgc-control-plane logs --all-containers --ignore-errors deployment/mgc-controller-manager -n multicluster-gateway-controller-system
          kubectl --context kind-mgc-control-plane logs --all-containers --ignore-errors deployment/mgc-policy-controller-manager -n multicluster-gateway-controller-system
          kubectl get managedzones -n multi-cluster-gateways mgc-dev-mz-aws -o yaml
          kubectl --context kind-mgc-control-plane -n multi-cluster-gateways wait --timeout=60s --for=condition=Ready managedzone/mgc-dev-mz-aws
          kubectl get managedzones -n multi-cluster-gateways mgc-dev-mz-gcp -o yaml
          kubectl --context kind-mgc-control-plane -n multi-cluster-gateways wait --timeout=60s --for=condition=Ready managedzone/mgc-dev-mz-gcp
```